tests/scanners/test.javascript.js

```diff
@@ -4,8 +4,8 @@ import { DEPRECATED_APIS, ESLINT_ERROR, ESLINT_RULE_MAPPING, TEMPORARY_APIS,
          VALIDATION_ERROR, VALIDATION_WARNING } from 'const';
 import JavaScriptScanner from 'scanners/javascript';
 import * as messages from 'messages';
-import * as rules from 'rules/javascript';
-import { apiToMessage, ignorePrivateFunctions, singleLineString } from 'utils';
+import ruleConfig from 'rules/javascript';
```

Comment:
I think you could do `import { rules } from 'rules/javascript';` and then you could just do `Object.keys(rules)`